test/project_ident_test.exs

```diff
@@ -11,7 +11,7 @@ defmodule ProjectIdentTest do
 
     mix_type = %ProjectType{name: :mix, path: "**", files: ["mix.exs,mix.lock"]}
     python_type = %ProjectType{name: :python, path: "**", files: ["setup.py,*requirements.txt*"]}
-    node_type = %ProjectType{name: :node, path: "**", files: ["package*.json"]}
+    node_type = %ProjectType{name: :node, path: "**", files: ["package*.json,yarn.lock"]}
```

Comment:
Is there a case where a project might have both?  Can't imagine, but also not positive.